model/dc/registry.yaml

```diff
@@ -0,0 +1,65 @@
+groups:
+  - id: registry.dc
+    type: attribute_group
+    display_name: Datacenter Attributes
+    brief: >
+      Generic DataCenter infrastructure tagging.
+    attributes:
+      - id: dc.name
+        type: string
+        stability: experimental
+        brief: 'Name of datacenter.'
+        examples: ['dc-name']
+      - id: dc.provider.type
+        type: string
+        stability: experimental
+        brief: 'Type of datacenter'
+        examples: ['cloud', 'colocation', 'internal']
+      - id: dc.device.type
+        type: string
+        stability: experimental
+        brief: 'Type of device used.'
+        examples: ['server', 'chassis', 'network', 'pdu', 'switch', 'patch-panel']
+      - id: dc.site
+        type: string
+        stability: experimental
+        brief: 'Name of the datacenter site'
```

Comment:
Can you outline more what this means and how to generate this field?  Can I discover these from within a datacenter or do I need to look it up?